app/brave_generated_resources.grd

```diff
@@ -396,6 +396,10 @@ By installing this extension, you are agreeing to the Google Widevine Terms of U
       <message name="IDS_SETTINGS_BRAVE_ON_EXIT" desc="Clear Browsing Data dialog On exit tab label">
         On exit
       </message>
+      <!-- In-page Translation -->
+      <message name="IDS_BRAVE_TRANSLATE_BUBBLE_BEFORE_TRANSLATE_TITLE" desc="Title text for the translate bubble when asking to translate a page.">
+        Translate this page? This will send all text on this page to Microsoft for translation.
```

Comment:
Do we plan to have a wiki that highlights the text will not be associated with any user identifiable information?

Reply:
cc: @diracdeltas

Reply:
not yet; maybe https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs could mention what data is sent in each of the proxied requests